topics/products/phpstorm/php_open_api.md

```diff
@@ -18,6 +18,36 @@
 
 ## Utility Classes
 
+### `com.jetbrains.php.lang.psi.elements.impl.PhpFilePathUtils`
```

Comment:
I'd use simple class name here and clarify FQN in first sentence

Reply:
Fixed.